packages/builders/src/messages/embed/Embed.ts

```diff
@@ -4,9 +4,6 @@ import {
 	colorPredicate,
 	descriptionPredicate,
 	embedFieldsArrayPredicate,
-	fieldInlinePredicate,
-	fieldNamePredicate,
-	fieldValuePredicate,
```

Comment:
Are those used anymore?

Reply:
they are used in [`embedFieldPredicate`](https://github.com/discordjs/discord.js/blob/main/packages/builders/src/messages/embed/Assertions.ts#L10-L14) which is used as `embedFieldsArrayPredicate` in `Embed`